cloud_experts_tutorials/cloud-experts-consistent-egress-ip.adoc

```diff
@@ -292,18 +326,18 @@ user-agent=curl/7.76.1
 BODY:
 -no body in request-
 ----
-
-You can safely exit the pod once you are done:
-
++
+. Exit the pod by running the following command:
++
 [source,terminal]
 ----
 $ exit
 ----
 
-=== Test pod egress
-
-Test the pod egress rule which was created previously. The following starts an interactive shell which allows you to run curl against the demo service:
+=== Testing the pod egress
 
+. Start an interactive shell and run `curl` against the demo service to test the pod egress rule:
```

Comment:
You use the `curl` command in the next step. Little confusing to introduce it here. Maybe explain the process before the steps?